package.json

```diff
@@ -49,7 +49,7 @@
     "@koa/cors": "^3.1.0",
     "@sentry/node": "^6.1.0",
     "@terra-money/terra.js": "^3.0.12",
-    "@terra-money/terra.proto": "^0.1.7",
+    "@terra-money/terra.proto": "^2.1.0",
```

Comment:
It could break things up since terra.proto to 2.x is for Terra 2.0

Reply:
👍 💯